lib/parser.dart

```diff
@@ -431,9 +467,13 @@ class SingleMapping extends Mapping {
       'names': names,
       'mappings': buff.toString()
     };
-    if (targetUrl != null) {
-      result['file'] = targetUrl;
+    if (targetUrl != null) result['file'] = targetUrl;
+
+    if (includeSourceContents) {
+      result['sourcesContent'] =
+          files.map((file) => file == null ? null : file.getText(0)).toList();
```

Comment:
I could be wrong about this because I'm still learning the best places for the question operator, but could "file == null ? null : file.getText(0)" be rewritten as something like:
file?.getText(0) I think that would work, but maybe it's less explicit and you're going for the more explicit route.

Reply:
You're totally right. Done.